xunit-autolabeler-v2/ast_parser/python/README.md

```diff
@@ -0,0 +1,15 @@
+# AST snippet parser - Python
+
+## What this does
+This tool extracts snippet data from Python snippets [such as these](https://github.com/googlecloudplatform/python-docs-samples), and serializes that data into a language-agnostic JSON format.
+
+That data can then be used by the "meta-parser" in `ast_parser/core` to match test results with their associated snippets/region tags.
+
+## Usage
+Run the following command to generate a `repo.json` file for a specific directory:
```

Comment:
Why is this file named `repo.json`?

Reply:
`repo.json` was based on the (now outdated) idea that each repo would have one of these files.

Maybe `snippets.json` or `snippet-data.json` would make more sense?

Reply:
(In any case - this shouldn't matter for this PR, as that file name isn't used if we don't add `python_bootstrap.py`)